### DISCUSSION TITLE
*option() and DVLOG(2)

### DISCUSSION THREAD

**Discussion by vhuber** [feelpp/feelpp#529] [Ideas]
May I add - for each function `*option()` the information :
- requested option,
- retrieved value,
  using DVLOG(2) ?

```
DVLOG(2) << "value for option " << name << " = " << opt << std::endl;
```

**Comment by prudhomm** [comment]
I don't understand what you want to do.

**Comment by vhuber** [comment]
I would like to add in the DVLOG stream information about the option, in order, for example, to produce with

``` c++
doption("gmsh.hsize");
```

the information about name (gmsh.hsize) and value.

**Comment by prudhomm** [comment]
@vincentchabannes what do you think about that ? this could be useful, perhaps with level 3 or 4 ?
